src/components/subComponents/AuditLogDownload.vue

```diff
@@ -28,11 +30,15 @@
     }, {immediate: true})
 
   async function getAuditlogMetadata(): Promise<void> {
-     await studyStore.getAuditLogMetadata(studyStore.studyId);
+
+     await studyStore.getAuditLogMetadata(studyStore.studyId)
+
   }
 
   function downloadCurrentAuditlog(): void {
+    isLoading.value = true
     studyStore.exportAuditLog(studyStore.studyId)
+      .then(() => isLoading.value = false);
```

Comment:
Bitte benutz finally, damit es bei Fehler auch reagiert
```suggestion
      .finally(() => isLoading.value = false);
```